src/components/Artist/Artist.test.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Artist from './Artist';
+
+jest.mock('react-router-dom', () => ({
```

Comment:
You got the mock test working! Nice! I couldn't no matter how much I fiddled with it.